### PR TITLE
Update docs: (i) docker, (ii) `run.sh`

### PR DESCRIPTION
**Pull request by joeflack4**
## Changes
Update docs
- Add: Docker prereqs
- Update: Running to utilize run.sh

## Related
- https://github.com/monarch-initiative/mondo-ingest/issues/446
- https://github.com/monarch-initiative/icd11/pull/3#discussion_r1496706806
- https://github.com/monarch-initiative/mondo-ingest/pull/447

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-On MacOS, (3) and (4) should be available without the need for installation.
+On macOS, (2) and (3) should be available without the need for installation.
 1. Python 3.9+
-2. `owltools`
-3. `make`
-4. Perl
+2. `make`
+3. Perl
+4. Docker
+5. Docker images
```
Comment (joeflack4) [inline README.md:9]: Docker dependencies

File: README.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+    - b. `docker pull obolibrary/odkfull:dev` 
 
 ## Setup
 1. Give permission to run Perl: `chmod +x ./bin/*.pl`
 2. Install Python dependencies: `pip install -r requirements.txt`
 
-## Running the ingest
-Run: `make all`
+## Running
+`sh run.sh make all`
```
Comment (joeflack4) [inline README.md:19]: `run.sh` usage

File: README.md [hunk 3]
```diff
@@ -1,15 +1,19 @@
 # MedGen ingest
 
 ## Prerequisites
-On MacOS, (3) and (4) should be available without the need for installation.
+On macOS, (2) and (3) should be available without the need for installation.
 1. Python 3.9+
```
Comment (matentzn) [inline README.md:5]: Why is python deeded? the only dependency should be ODK - nothing else?

Comment (matentzn) [reply README.md:5]: (and bash shell)

File: README.md [hunk 4]
```diff
@@ -1,15 +1,19 @@
...
-2. `owltools`
-3. `make`
-4. Perl
+2. `make`
+3. Perl
```
Comment (matentzn) [inline README.md:7]: Again, only ODK dependency should be needed.

File: README.md (hunk 3 above)
Comment (joeflack4) [reply README.md:5]: Oh, that's true. Well, usually projects don't do a good job of distinguishing "dev dependencies" very well. Python is needed for development (assuming you want to be able to use a debugger). If you want I can drop it from the list, though.

File: README.md (hunk 4 above)
Comment (joeflack4) [reply README.md:7]: Likewise same response as: https://github.com/monarch-initiative/medgen/pull/17#discussion_r1499798898

Actually, now that I remember, funny but when I asked Chris how to debug Perl, he laughed. But seriously though, the Perl there is complex enough to where debugging would be very helpful. I was making edits without the use of a debugger, but I do see that there are methods to debug perl ([example](https://perldoc.perl.org/perldebug)).

File: README.md (hunk 3 above)
Comment (matentzn) [reply README.md:5]: Decide by yourself!

Comment (joeflack4) [reply README.md:5]: Alright then, I'll add a note that these 2 are just dev dependencies, and I'll leave in the list!

**Comment by joeflack4** [comment]
I'm going to be making similar docs updates to other repositories. Merging this now as I think it has been approved, and this will make it easier for me to do the other PRs.

**Pull request merged**